src/status_im/contexts/wallet/events.cljs

```diff
@@ -439,3 +439,32 @@
                 :type     :negative
                 :text     (i18n/label :t/provider-is-down {:chains chain-names})
                 :duration 10000}]]])})))
+
+(rf/reg-event-fx
+ :wallet/save-address
+ (fn [_ [{:keys [address name color-id on-success on-error chain-short-names ens is-test]
```

Comment:
nit but I don't think we refer to it as `color-id` elsewhere. Can we change this to customization-color or like user-color?

Reply:
afaik customization-color is only for the color selected by the user for the profile, or is it for all colors that change the ui appearence?

Reply:
It's more a set of colors tbh, which this would fit into since it's the set of colors a user can set for their profile